src/components/ViewDao/DaoSidebar.tsx

```diff
@@ -55,7 +55,9 @@ class DaoSidebarComponent extends React.Component<IProps, null> {
               </li>
               <li>
                 <NavLink activeClassName={css.selected} to={"/dao/" + dao.address + "/redemptions/"}>
+{/*                TODO: Make this notification work!
```

Comment:
is there an issue for this?

Reply:
https://github.com/daostack/alchemy/issues/572